provider/README.md

```diff
@@ -268,11 +268,14 @@ The `/trips` API should allow querying trips with the following query parameters
 | --------------- | ------ | --------------- |
 | `end_time` | `YYYY-MM-DDTHH`, an [ISO 8601](https://en.wikipedia.org/wiki/ISO_8601) extended datetime representing an UTC hour between 00 and 23. | All trips with an end time occurring within the hour. For example, requesting `end_time=2019-10-01T07` returns all trips where `2019-10-01T07:00:00 <= trip.end_time < 2019-10-01T08:00:00` UTC. |
 
-If the data does not exist or the hour has not completed, `/trips` shall return a `404 Not Found` error.
+For hours that have not yet completed or during which the provider was not operating
+in a municipality `/trips` shall return a `404 Not Found` error.
+`/trips` shall return an empty array of trips for
```

Comment:
`/trips` must return a Payload with an an empty array of trips `{ "trips": [] }` for

Reply:
Yes.